SASS/signup.css

```diff
@@ -57,9 +57,35 @@ body {
 }
 
 .footer {
+  margin-bottom: 0;
+  width: 100%;
+  background: #60a06D;
+}
+.footer p {
   text-align: center;
+  color: white;
+  font-size: 1.2rem;
+  padding: 10px;
 }
 
+@media (max-width: 600px) {
+  .container {
```

Comment:
You might want to try adjusting the width on the container class at mobile width, as when the images disappear the everything's kinda pushed to the left of the page.

You could maybe add a media query to the welcome class and text-align center, as well, so everything's lined up. I'm not great with styling, though, maybe see if Justin Stevens has any thoughts.